__tests__/app.test.js

```diff
@@ -408,4 +408,32 @@ describe("app", () => {
             })
         })
     })
+
+    describe("DELETE /api/comments/:comment_id", () => {
+
+        test("should delete restaurant with corresponding id", () => {
+            return request(app).delete("/api/comments/6").expect(204).then(() => {
```

Comment:
the second query isn't really necessary here... What would our then block expect to recieve upon a successful delete? Maybe we can use this to confirm the delete has been successful